packages/widget_toolkit/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [0.0.1-dev7]
+- Documentation improvements
+### Breaking Changes:
+- Removed params: `showChangeLanguageBottomSheet(iconRight, errorPanelIcon)`
+- Added parameter `showChangeLanguageBottomSheet(bool? hasLeftIcon, Widget Function(SelectedLanguageModel model, bool loading, BuildContext context)? itemBuilder)`
```

Comment:
Since we have added the `itemBuilder` why we need the `hasLeftIcon` param ?

Reply:
I added the `hasLeftIcon`, because we already had an icon configured to be loaded on the left of the text, but we were not using and now the user can use it if he/she wants without the need to use the `itemBuilder`. But I can remove the `hasLeftIcon` parameter and this part from `SelectLanguageItem`:
 ```
 if (hasLeftIcon ?? false)
            DynamicIcon(
              context.languagePickerTheme.leftInfoCircleIcon,
              color: _getTextColor(context),
            ),
```
Should I delete it

Reply:
Please add golden tests so that I can see what you mean :)

https://github.com/Prime-Holding/widget_toolkit/tree/fix/language_picker_api_improvements/packages/widget_toolkit/test/lib_language_picker

Looking at the gif, we use some icons and I'm not sure what the impact will be
https://raw.githubusercontent.com/Prime-Holding/widget_toolkit/master/packages/widget_toolkit/doc/assets/language-picker.gif

Reply:
I have added a golden test for the success state without using `hasLeftIcon` and one golden test with using `hasLeftIcon` where the additional icon is visible

Reply:
https://github.com/Prime-Holding/widget_toolkit/blob/fix/language_picker_api_improvements/packages/widget_toolkit/test/lib_language_picker/view/goldens/light_theme/language_picker_has_left_icon.png